packages/client/src/workflow-client.ts

```diff
@@ -806,13 +806,15 @@ export class WorkflowClient extends BaseClient {
     waitForStage: WorkflowUpdateStage,
     input: WorkflowStartUpdateInput
   ): Promise<WorkflowStartUpdateOutput> {
+    waitForStage = waitForStage >= WorkflowUpdateStage.ACCEPTED ? waitForStage : WorkflowUpdateStage.ACCEPTED;
```

Comment:
Is this a bug fix -- I was allowing clients to submit `Admitted`? Thanks if so! (I should add test coverage for it.)